src/app/infobox/infobox.component.ts

```diff
@@ -41,11 +48,10 @@ export class InfoboxComponent implements OnInit {
         }
 
     });
-
-
   }
 
   ngOnInit() {
+    this.synthesis.speak(this.results[0]);
```

Comment:
we won't receive results immediately when a component is initialised. you have to call this function only after we receive results from the server.